### DISCUSSION TITLE
Your memory game doesn't work any more

### DISCUSSION THREAD

**Discussion by yyyasmin** [community/community#31153] [Other Feature Feedback, Questions, & Ideas]
I like your memory game very much. It use to work but not any more' probably due to versions issues. would you update it so it will work with later versions ( I guess of python and other stuff). 
If not, would you tell me how to fix it ? 
Thankyou, Regards
yasminamran@gmail.com

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, but from the text I can't guess what. Anyway, in that case you should open an issue *in the repository for that software* so the developers see it. They'll probably need a bit more information on how it doesn't work. :sweat_smile:
